cmd/crio/main.go

```diff
@@ -444,56 +474,66 @@ func main() {
+			EnvVar: "CONTAINER_DEFAULT_CAPABILITIES",
 		},
 		cli.StringSliceFlag{
 			Name:  "default-sysctls",
 			Usage: fmt.Sprintf("sysctls to add to the containers (default: %q)", defConf.DefaultSysctls),
 		},
 		cli.StringSliceFlag{
 			Name:  "default-ulimits",
-			Usage: fmt.Sprintf("ulimits to apply to conatainers by default (name=soft:hard) (default: %q)", defConf.DefaultUlimits),
+			Usage: fmt.Sprintf("ulimits to apply to containers by default (name=soft:hard) (default: %q)", defConf.DefaultUlimits),
 		},
 		cli.BoolFlag{
-			Name:  "profile",
-			Usage: "enable pprof remote profiler on localhost:6060",
+			Name:   "profile",
+			Usage:  "enable pprof remote profiler on localhost:6060",
+			EnvVar: "CONTAINER_PROFILE",
 		},
 		cli.IntFlag{
-			Name:  "profile-port",
-			Value: 6060,
-			Usage: "port for the pprof profiler",
+			Name:   "profile-port",
+			Value:  6060,
+			Usage:  "port for the pprof profiler",
+			EnvVar: "CONTAINER_PROFILE_PORT",
 		},
 		cli.BoolFlag{
-			Name:  "enable-metrics",
-			Usage: "enable metrics endpoint for the server on localhost:9090",
+			Name:   "enable-metrics",
+			Usage:  "enable metrics endpoint for the server on localhost:9090",
+			EnvVar: "CONTAINER_ENABLE_METRICS",
 		},
 		cli.IntFlag{
-			Name:  "metrics-port",
-			Value: 9090,
-			Usage: "port for the metrics endpoint",
+			Name:   "metrics-port",
+			Value:  9090,
+			Usage:  "port for the metrics endpoint",
+			EnvVar: "CONTAINER_METRICS_PORT",
 		},
 		cli.BoolFlag{
-			Name:  "read-only",
-			Usage: fmt.Sprintf("setup all unprivileged containers to run as read-only (default: %t)", defConf.ReadOnly),
+			Name:   "read-only",
+			Usage:  fmt.Sprintf("setup all unprivileged containers to run as read-only (default: %t)", defConf.ReadOnly),
+			EnvVar: "CONTAINER_READ_ONLY",
 		},
 		cli.StringFlag{
-			Name:  "bind-mount-prefix",
-			Usage: fmt.Sprintf("specify a prefix to prepend to the source of a bind mount (default: %q)", defConf.BindMountPrefix),
+			Name:   "bind-mount-prefix",
+			Usage:  fmt.Sprintf("specify a prefix to prepend to the source of a bind mount (default: %q)", defConf.BindMountPrefix),
+			EnvVar: "CONTAINER_BIND_MOUNT_PREFIX",
 		},
 		cli.StringFlag{
-			Name:  "uid-mappings",
-			Usage: fmt.Sprintf("specify the UID mappings to use for the user namespace (default: %q)", defConf.UIDMappings),
-			Value: "",
+			Name:   "uid-mappings",
+			Usage:  fmt.Sprintf("specify the UID mappings to use for the user namespace (default: %q)", defConf.UIDMappings),
+			Value:  "",
+			EnvVar: "CONTAINER_UID_MAPPINGS",
 		},
 		cli.StringFlag{
-			Name:  "gid-mappings",
-			Usage: fmt.Sprintf("specify the GID mappings to use for the user namespace (default: %q)", defConf.GIDMappings),
-			Value: "",
+			Name:   "gid-mappings",
+			Usage:  fmt.Sprintf("specify the GID mappings to use for the user namespace (default: %q)", defConf.GIDMappings),
+			Value:  "",
+			EnvVar: "CONTAINER_GID_MAPPINGS",
```

Comment:
It would be nice to have the variables in the active and deprecated sections listed alphabetically by name.